spi_nand_flash/src/spi_nand_oper.c

```diff
@@ -83,7 +95,8 @@ esp_err_t spi_nand_read(spi_device_handle_t device, uint8_t *data, uint16_t colu
         .address = column,
         .miso_len = length,
         .miso_data = data,
-        .dummy_bits = 8
+        .dummy_bits = 8,
+        .flags = SPI_TRANS_DMA_BUFFER_ALIGN_MANUAL,
```

Comment:
Perhaps we can just guard this part with an IDF version check, to avoid raising the minimal supported IDF version?